test/database.ts

```diff
@@ -1590,7 +1589,7 @@ describe('Database', () => {
     it('should send labels correctly', done => {
       const labels = {a: 'b'};
       const options = {a: 'b', labels};
-      const originalOptions = extend(true, {}, options);
+      const originalOptions = Object.assign({}, options, labels);
```

Comment:
This is technically different.  The `true` flag passed into the first argument of the `extend` method signals a *deep* object copy.  `Object.assign` only does shallow clones. For the purposes of the tests it may not matter, just be aware this is a behavior change, and not one we should make make in the `src/` dir without heavy, heavy testing.